xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.512.xml

```diff
@@ -104,18 +106,46 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+           <ab>
+              <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>T EI<gap reason="lost" extent="unknown" unit="character"/>
+              <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/>cun<gap reason="lost" extent="unknown" unit="character"/>
+              <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/>et 
+                 <persName><name type="praenomen" key="Quintus"><expan><abbr>Q</abbr><ex cert="low">uintus</ex></expan></name> 
+                 <name>Me<gap reason="lost" extent="unknown" unit="character"/></name></persName>
+              <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">pi</supplied>entissum 
+                  <gap reason="lost" extent="unknown" unit="character"/>
+              <lb n="5"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">d</supplied>omo 
+                  Tici<supplied reason="lost">no <expan><abbr>vix</abbr><ex>it</ex></expan> <expan><abbr>ann</abbr><ex>os</ex></expan></supplied>
+              <lb n="6"/><num xml:lang="lat" value="33">XXXIII</num> mi<supplied reason="lost">litavit <expan><abbr>ann</abbr><ex>os</ex></expan></supplied>
+              <lb n="7"/><num xml:lang="lat" value="13">XIII</num> <expan><abbr>co<supplied reason="lost">h</supplied></abbr><ex>ortis</ex></expan> 
+                  <gap reason="lost" extent="unknown" unit="character"/>
+              <lb n="8"/>hic s<supplied reason="lost">itus est</supplied>.
+           </ab>
+        </div>
         
         
                 <div type="translation">
-                    <p/>
+                    <p>
+                       [---]T EI[---]
+                       [---]cun[---]
+                       and Quintus Me[---]
+                       [---] most pious [---]
+                       [---] from the town of Ticinum, lived for 33 years and soldiered for 
+                       13 years, from the cohort of [---],
+                       lies here.
+                    </p>
                 </div>
             
         
         
                 <div type="commentary">
-                    <p/>
+                    <p>
+                       Lines 1 and 2 - recorded in CIL, now missing. Likely contained name of dedicator.
```

Comment:
This means that they should be marked up as `<supplied reason="lost" evidence="previouseditor">`